client/state/ui/editor/contact-form/test/reducer.js

```diff
@@ -2,21 +2,26 @@
  * External dependencies
  */
 import { assert } from 'chai';
+import deepFreeze from 'deep-freeze';
 
 /**
  * Internal dependencies
  */
+import i18n from 'lib/mixins/i18n';
 import reducer from '../reducer';
 import { CONTACT_FORM_DEFAULT, CONTACT_FORM_FIELD_TYPES } from '../constants';
 import {
-	EDITOR_CONTACT_FORM_LOAD_FORM,
-	EDITOR_CONTACT_FORM_ADD_DEFAULT_FIELD,
-	EDITOR_CONTACT_FORM_REMOVE_FIELD,
-	EDITOR_CONTACT_FORM_CLEAR_FORM
+	EDITOR_CONTACT_FORM_CLEAR,
+	EDITOR_CONTACT_FORM_LOAD,
+	EDITOR_CONTACT_FORM_FIELD_ADD,
+	EDITOR_CONTACT_FORM_FIELD_REMOVE,
+	EDITOR_CONTACT_FORM_FIELD_UPDATE,
+	EDITOR_CONTACT_FORM_SETTINGS_UPDATE
 } from 'state/action-types';
 
-describe( "editor's contact form state reducer", () => {
+i18n.initialize();
```

Comment:
we don't currently, but we can add it to the global before suite. It just moved, so this would have to be rebased to pick it up.